docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -633,6 +751,24 @@ These rules apply to ``filter_environment`` and ``mask_environment``.
 
     ``mask`` can contain conditional expressions such as ``IF`` or ``CASE``, which achieves conditional masking.
 
+Function rules
+^^^^^^^^^^^^^^
+
+Each function rule is composed of the following fields:
+
+* ``user`` (optional): regular expression to match against user name.
+  Defaults to ``.*``.
+* ``group`` (optional): regular expression to match against group names.
+  Defaults to ``.*``.
+* ``schema`` (optional): regular expression to match against schema name.
+  Defaults to ``.*``.
+* ``function`` (optional): regular expression to match against function names.
+  Defaults to ``.*``.
+* ``privileges`` (required): zero or more of ``EXECUTE``, ``GRANT_EXECUTE``.
+* ``function_kinds`` (required): one or more of ``AGGREGATE``, ``SCALAR``,
+  ``TABLES``, ``WINDOW``. When a user defines a rule for ``AGGREGATE``, ``SCALAR``
+  or ``WINDOW`` functions, the ``catalog`` and ``schema`` fields are disallowed.
```

Comment:
should we add why ... 

because these are globally defined functions 

or something like that

Reply:
Ok, added explanation why